src/ethereum/frontier/genesis.py

```diff
@@ -18,16 +18,16 @@
 from typing import Dict, cast
 
 from ethereum.base_types import U256, Bytes, Bytes8, Uint, slotted_freezable
-from ethereum.frontier.eth_types import Address
-
-from .utils.hexadecimal import (
-    hex_to_address,
+from ethereum.utils.hexadecimal import (
     hex_to_bytes,
     hex_to_bytes8,
     hex_to_u256,
     hex_to_uint,
 )
 
+from .eth_types import Address
```

Comment:
I understand the intent that modules outside the frontier related stuff shouldn't import the frontier stuff. But the genesis seems that it can be uncoupled from the frontier stuff as a standalone entity.

Further I think that some of the artefacts present in `eth_types` (like `Address`), utils should be moved out of `frontier` module as they don't seem to be tightly coupled with `frontier`. Let me know what you think about this.

Reply:
`Address` does seem pretty generic, and my only argument for putting it in the hardfork is because of the recent address space extension talks.

This PR is just a quick bandaid to get things "good enough" for now.

Reply:
Sounds good, thanks.